src/css/base.scss

```diff
@@ -4,3 +4,7 @@ $primary-background: pink;
 body {
   background-color: $primary-background;
 }
+
+.hidden {
+  display: none !important
+}
```

Comment:
Does this need the !important? Just curious since I have seen that they recommend we use it sparingly.